app/controllers/outfits_controller.rb

```diff
@@ -41,12 +36,14 @@ def destroy
   private
 
   def outfit_params
-    # params.require(:outfit).permit!
     params.fetch(:outfit, {}).permit!
   end
 
-  # def outfits_item_params
-    # params.require(:outfits_item).permit(:outfit_id, :item_id)
-  # end
-
+  def create_new_outfit_with_items
```

Comment:
@whatthemandy 
Before filters are there to reduce repetition of code, since this is happening only in the `#create` action, it may not be the best refactor. Instead of pulling this into a before action, It may be better to put this into an object that can coordinate the creation of the outfit, and even wrap the entire thing in a transaction, so that it either all saves, or none of it saves.

ActiveRecord Transactions: http://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html